test/redix_test.exs

```diff
@@ -6,8 +6,13 @@ defmodule RedixTest do
 
   import Redix.TestHelpers
 
+  @redis_host (System.get_env("REDIS_HOST") || "localhost") |> String.to_char_list
+  @redis_port System.get_env("REDIS_PORT") || 6379
+  @resis_host_and_port "#{@redis_host}:#{@redis_port}"
```

Comment:
Agreed, and since we're here we could probably remove the `redis_` prefix of this module attribute since I think host/port is clear enough already :)